### ISSUE TITLE
responsive font style

### ISSUE THREAD

**Issue by Melkor333** [nemanjan00/uniread#14]
Excuse my burst of feature requests, but I just had the idea that some kind of responsiveness could enhance the readability. for example make the name of a color in that color, quotes in italic, if an exclamation mark is at the end of a sentence, make the sentence bold, etc.

I don't really know if this enhances or reduces reading/comprehension speed but it may be worth a try... What do you think about that idea?

**Comment by nemanjan00** [comment]
Thanks for sharring! :)

**Comment by nemanjan00** [comment]
Implemented

**Comment by Melkor333** [comment]
LLM acceleration sometimes feels like magic 😄

**Comment by nemanjan00** [comment]
It is not as much as acceleration as I would not have time at all to work on it without AI

**Comment by Melkor333** [comment]
Yeah true, for these kinds of projects it can literally just do the job for you

**Comment by nemanjan00** [comment]
Not only that, it does other things too for me, which gives me at least time to monitor it
